pmm-app/src/pmm-dbaas/components/DBCluster/PSMDB.service.ts

```diff
@@ -32,6 +32,29 @@ export class PSMDBService extends DBClusterService {
     );
   }
 
+  updateDBCluster(dbCluster: DBCluster): Promise<void | DBClusterPayload> {
+    const toAPI = (cluster: DBCluster) => ({
```

Comment:
Don't forget to remove this toAPI methods like you did in other branch.

Reply:
done